src/main/assembly/dist/cfg/application.properties

```diff
@@ -3,5 +3,5 @@ bag-store.uuid-component-sizes=2,30
 bag-store.bag-file-permissions=r--r--r--
 staging.base-dir=/srv/dans.knaw.nl/stage
 daemon.http.port=20110
-daemon.external-base-uri=http://localhost:20110/
+daemon.external-base-uri=http://test.dans.knaw.nl/
```

Comment:
@rvanheest : please revert this, as `http://localhost:20110` is a more plausible "reasonable default".